phpcs.xml

```diff
@@ -1,20 +1,17 @@
 <?xml version="1.0"?>
 <ruleset name="The Events Calendar Coding Standards">
-	<rule ref="TribalScents"></rule>
 	<rule ref="WordPress-VIP-Go"></rule>
 	<rule ref="WordPress">
 		<exclude name="WordPress.Files.FileName"/>
-	</rule>
-	<rule ref="WordPress-Extra"></rule>
-	<rule ref="WordPress-Docs"></rule>
-	<rule ref="WordPress-Core"></rule>
-
-	<rule ref="Generic">
 		<exclude name="Generic.Arrays.DisallowShortArraySyntax"/>
 	</rule>
+	<rule ref="TribalScents"></rule>
 
 	<!--exclude the rule for violation of direct DB calls as some have no alternative-->
 	<rule ref="WordPress.DB.DirectDatabaseQuery">
 		<exclude-pattern>src/Test.php</exclude-pattern>
 	</rule>
+
+	<exclude-pattern>*/tests/_support/_generated/*</exclude-pattern>
```

Comment:
Should we not exclude the whole tests folder?

Reply:
Perhaps. Though, it'd be nice if our tests also conformed to coding standards.

Reply:
Then, can we skip, for tests, the commenting ones?
Tests will have weird doc formats and tags.

Reply:
Added here: https://github.com/moderntribe/the-events-calendar/pull/3070/files#diff-6b216d983cf0a62e9c648934c5687456R5

Reply:
I'm sure more will pop up as we use this, but fine for the time being.